### PR TITLE
Upgrade to Spring 6 and to Jakarta namespaces

### PR DESCRIPTION
**Pull request by jkosternl**
Not ready yet, these are the current remaining issues:
3. Ladybug & Echo2 investigation needed and upgraded Spring 6 deps with Jakarta namespace

### CODE REVIEW

**Comment by nielsm5** [comment]
So it looks like Spring is implementing `(smart)Lifecycles` to certain beans to allow jvm pauze/resume behavior. See
https://github.com/spring-projects/spring-framework/issues/30831, the newly introduced `acceptTasksAfterContextClose` could help, and/or we should prioritize/order the lifecycles stop order.

File: test-integration/pom.xml [hunk 1]
```diff
@@ -41,8 +41,9 @@
 		</dependency>
 
 		<dependency>
-			<groupId>org.apache.geronimo.specs</groupId>
-			<artifactId>geronimo-jms_1.1_spec</artifactId>
+			<groupId>jakarta.management.j2ee</groupId>
+			<artifactId>jakarta.management.j2ee-api</artifactId>
+			<version>1.1.4</version>
```
Comment (nielsm5) [inline test-integration/pom.xml:46]: jms is niet j2ee :D

Comment (jkosternl) [reply test-integration/pom.xml:46]: Ah ja oeps, iets te snel vervangen. Ga ik fixen.

File: cmis/pom.xml [hunk 2]
```diff
@@ -29,14 +29,10 @@
 		</dependency>
 
 		<dependency>
-			<groupId>org.apache.chemistry.opencmis</groupId>
+			<groupId>com.kgs-software</groupId>
```
Comment (tnleeuw) [inline cmis/pom.xml:32]: Ik neem aan dat deze dependency-change nodig is om CMIS compatible te maken met de nieuwe Servlet API en Jakarta namespaces.

**Review by nielsm5 — APPROVED** [review]
groen, groen & groen

**Review by tnleeuw — APPROVED** [review]

**Pull request merged**